dependencies/alfont/CMakeLists.txt

```diff
@@ -45,14 +45,14 @@ target_include_directories(alfont
 
 target_link_libraries(alfont
   PUBLIC
-    allegro
+    alleg
 )
 
 # Compile & link options are taken from the Makefile. 
 target_compile_options(alfont
   PRIVATE
     "-O2"
-    "-march=pentium"
+#    "-march=pentium"
```

Comment:
This didn't work?
TBH, I don't really care. I copied this from the makefile of alfont, but to me it seems weird to set an architecture.